examples/createtool/createtool.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Cindy JS</title>
+        <style>
+            body {
+                margin: 0;
+                padding: 0;
+            }
+
+            button {
+                background-color: #FFFFFF;
+                border: 1px solid #CCCCCC;
+            }
```

Comment:
More interesting buttons are possible with this instead of the button style used here :smile:
```css
            #toolbar button {
                padding: 0px 0px 0px 0px;
                line-height: 0px;
                border:0px none red;
                padding: 1px;
                margin: 0px;
            }
            #toolbar img {
                background-image: url(../../build/js/images/Empty.png);
                padding: 2px;
            }
```
![empty](https://cloud.githubusercontent.com/assets/15895776/14582335/0dd96e96-03bf-11e6-94d1-8577d1b7bc1a.png)